src/test/java/org/prebid/server/bidder/invibes/InvibesBidderTest.java

```diff
@@ -291,7 +295,55 @@ public void makeHttpRequestsShouldCreateInvibesBidRequestWithCorrectParams() thr
         assertThat(result.getErrors()).isEmpty();
         assertThat(result.getValue()).hasSize(1)
                 .extracting(HttpRequest::getPayload)
-                .containsOnly(expectedRequest);
+                .containsExactly(expectedRequest);
+    }
+
+    @Test
+    public void makeHttpRequestsShouldCreateInvibesAmpBidRequestWithCorrectParams() throws JsonProcessingException {
+        // given
+        final BidRequest bidRequest = givenBidRequest(
+                bidRequestBuilder -> bidRequestBuilder
+                        .device(Device.builder().w(DEVICE_W).h(DEVICE_H).build())
+                        .user(User.builder().buyeruid(BUYER_UID).build())
+                        .site(Site.builder().page(PAGE_URL).build())
+                        .ext(ExtRequest.of(
+                                ExtRequestPrebid.builder().amp(ExtRequestPrebidAmp.of(Collections.emptyMap())).build()
+                        )),
+                impBuilder -> impBuilder.banner(Banner.builder().h(BANNER_H).w(BANNER_W).build()));
+
+        // when
+        final Result<List<HttpRequest<InvibesBidRequest>>> result = invibesBidder.makeHttpRequests(bidRequest);
+
+        // then
+        final Map<String, InvibesPlacementProperty> properties = new HashMap<>();
+        properties.put(FIRST_PLACEMENT_ID, InvibesPlacementProperty.builder()
+                .formats(Collections.singletonList(Format.builder().w(BANNER_W).h(BANNER_H).build())).build());
+
+        final InvibesBidParams invibesBidParams = InvibesBidParams.builder()
+                .placementIds(Collections.singletonList(FIRST_PLACEMENT_ID))
+                .bidVersion(BID_VERSION)
+                .properties(properties)
+                .build();
+
+        final InvibesBidRequest expectedRequest = InvibesBidRequest.builder()
+                .bidParamsJson(mapper.writeValueAsString(invibesBidParams))
+                .isTestBid(true)
+                .location(PAGE_URL)
+                .isAmp(true)
+                .gdpr(true)
+                .gdprConsent(StringUtils.EMPTY)
+                .invibBVLog(true)
+                .videoAdDebug(true)
+                .lid(BUYER_UID)
+                .bvid("test")
+                .width(String.valueOf(DEVICE_W))
+                .height(String.valueOf(DEVICE_H))
+                .build();
+
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1)
```

Comment:
Should we check the whole request or only `isAmp` flag?

Reply:
I opted for checking the entire request as it should stay the same with only the AMP flag changed